### DISCUSSION TITLE
Support f2fs mount

### DISCUSSION THREAD

**Discussion by TheUtopian** [util-linux/util-linux#2532] [Ideas]
I'd like to see f2fs support

**Comment by t-8ch** [comment]
Could you elaborate what you deem missing?

This should already work.

**Reply by TheUtopian** [reply]
Sorry, you're right, I had confused busybox mount with util-linux.

**Reply by t-8ch** [reply]
No worries.

FYI There is nothing special in util-linux mount about f2fs.
So it should also work from busybox mount.
(It does for me)

**Reply by AndreiCherniaev** [reply]
I met interesting issue with f2fs, can you help me please? Description here https://web.archive.org/web/20241108082547/https://stackoverflow.com/questions/79168984/how-to-mount-f2fs-with-option-compress-algorithm
@t-8ch @TheUtopian

**Reply by t-8ch** [reply]
@AndreiCherniaev your issue seems to be with busybox mount which is unrelated to util-linux mount.
